test/uk/gov/hmrc/economiccrimelevyreturns/controllers/CheckYourAnswersControllerSpec.scala

```diff
@@ -16,31 +16,40 @@
 
 package uk.gov.hmrc.economiccrimelevyreturns.controllers
 
+import com.danielasfregola.randomdatagenerator.RandomDataGenerator.derivedArbitrary
+import play.api.mvc.Result
 import play.api.test.Helpers._
 import uk.gov.hmrc.economiccrimelevyreturns.base.SpecBase
+import uk.gov.hmrc.economiccrimelevyreturns.models.EclReturn
 import uk.gov.hmrc.economiccrimelevyreturns.views.html.CheckYourAnswersView
 import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryList
 
+import scala.concurrent.Future
+
 class CheckYourAnswersControllerSpec extends SpecBase {
 
   val view: CheckYourAnswersView = app.injector.instanceOf[CheckYourAnswersView]
 
-  val controller = new CheckYourAnswersController(
-    messagesApi,
-    fakeAuthorisedAction,
-    fakeDataRetrievalAction(),
-    mcc,
-    view
-  )
+  class TestContext(eclReturnData: EclReturn) {
```

Comment:
name the parameter eclReturn rather than eclReturnData (to be consistent with elsewhere)?